subsys/debug/tracing/include/tracing_cpu_stats.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _TRACE_CPU_STATS_H
+#define _TRACE_CPU_STATS_H
+#include <kernel.h>
+#include <kernel_structs.h>
+#include <init.h>
+#include <misc/printk.h>
```

Comment:
why do you need printk here?

Reply:
Currently printk is used to periodically display information about CPU usage.

Reply:
it should be included where it is being used.

Reply:
done